test/functional/feature_taproot.py

```diff
@@ -1389,7 +1388,7 @@ def test_spenders(self, node, spenders, input_counts):
             amount = sum(utxo.output.nValue for utxo in input_utxos)
             fee = min(random.randrange(MIN_FEE * 2, MIN_FEE * 4), amount - DUST_LIMIT)  # 10000-20000 sat fee
             in_value = amount - fee
-            tx.vin = [CTxIn(outpoint=utxo.outpoint, nSequence=random.randint(min_sequence, 0xffffffff)) for utxo in input_utxos]
+            tx.vin = [CTxIn(outpoint=utxo.outpoint, nSequence=random.randint(0xffffffff-2, 0xffffffff)) for utxo in input_utxos]
```

Comment:
@sipa can you confirm that changing this test like this is OK -- perhaps you'd prefer to preserve the test as is but  set is_standard_tx to false when appropriate?